Skip tests requiring pytest-relaxed since it was removed from the tree due to
breaking pytest.

--- paramiko-2.6.0/tests/test_client.py
+++ paramiko-2.6.0/tests/test_client.py
@@ -33,7 +33,7 @@
 import weakref
 from tempfile import mkstemp
 
-from pytest_relaxed import raises
+#from pytest_relaxed import raises
 from mock import patch, Mock
 
 import paramiko
@@ -684,7 +684,8 @@
 
     # TODO: more granular exception pending #387; should be signaling "no auth
     # methods available" because no key and no password
-    @raises(SSHException)
+    #@raises(SSHException)
+    @unittest.skip("skip tests requiring pytest-relaxed")
     def test_passphrase_kwarg_not_used_for_password_auth(self):
         # Using the "right" password in the "wrong" field shouldn't work.
         self._test_connection(passphrase="pygmalion")
@@ -705,7 +706,8 @@
             password="television",
         )
 
-    @raises(AuthenticationException)  # TODO: more granular
+    #@raises(AuthenticationException)  # TODO: more granular
+    @unittest.skip("skip tests requiring pytest-relaxed")
     def test_password_kwarg_not_used_for_passphrase_when_passphrase_kwarg_given(  # noqa
         self
     ):
